### ISSUE TITLE
Speed Unit configuration?

### ISSUE THREAD

**Issue by xeviff** [pedrocesar-ti/internet-speedtest-docker#25]
Hi!
Thanks for this amazing docker stack.
I was wondering if we can change easily the speed units from MB/s to Mbps. As I saw in your screenshots, you use Mbps units but in my Grafana the results are shown in MB/s.
Thanks for advance! and a great regard

**Comment by pedrocesar-ti** [comment]
Hi @xeviff, I just merged a small PR to fix the unit presented in Grafana, the MBs was probably introduced when we moved Grafana to an external configuration file, that's why on the old screenshot shows Mbps. I just fixed so if you pull the image again should show correctly.

**Comment by xeviff** [comment]
Hello again @pedrocesar-ti !
I came back here because after the units update, my speed always appear like **10 Mbps download and 5 Mbps upload** (see image). 

![image](https://user-images.githubusercontent.com/73612508/136666195-a4f4a411-ff99-435d-b3db-cfa03ad82254.png)

However, I'm pretty sure that it's not correct or something is boycotting the results. Let me show you why: 
1. Your docker (which I love despite all) is executed from my NAS wich has two gigabit ethernet cables connected directly to the router, making a **2 Gbps connection** (see image)

![image](https://user-images.githubusercontent.com/73612508/136666300-d2521e98-b651-4b2f-a2f9-483ae8395007.png)

2.  In other hand I have a laptop, connected by Wifi with about **300 Mbps** 

![image](https://user-images.githubusercontent.com/73612508/136666337-c24563eb-2dbb-46af-bcac-d63e4b618c44.png)

3. I do a speed test from my laptop through the speedtest.net webapp. I should get wrose speed, right? Well, what I'm getting is around **100 Mbps download and 300 Mbps upload** (ten times more???). See image:

![image](https://user-images.githubusercontent.com/73612508/136666581-86f23f1e-54f3-4c39-9b96-4bdc7b0e752a.png)

Has it any sense? 
Like you, I wanted to show my reports to ISP through this docker (since I contracted 1 GB and actually I get around 100 Mbps) but right now it doesn't have a reliable information.
I'm wondering if you're imagining what can cause this disorder.

Best regards.
Xavi
